### DISCUSSION TITLE
is there anyone tried backpack with multitenant implementation ?

### DISCUSSION THREAD

**Discussion by siberfx** [Laravel-Backpack/community-forum#310] [Q&A (Help)] [answered]
I tried to make the implementation for a multitenant system, but I didnt achieve it.. if there are anyone who tried/achieve may show up ?

**Comment by quiquegarcia** [comment, marked as answer]
Have you tried: https://spatie.be/docs/laravel-multitenancy/v1/introduction ?

Maybe implementing that and then in Admin Panel switch between domain/tenant if the user is an admin or simply do a ```Tenant::checkCurrent()``` that is stored in the user profile.

**Comment by pxpm** [comment]
I had https://github.com/tenancy/multi-tenant working in some 3.6 project that I started but never finished. I don't have the project here by hand, but if I recall correctly it was something along the lines that @quiquegarcia suggested. Using a custom middleware and some other stuff. I created models/controllers etc for Websites and Hostnames, different database per tenant etc.

I had an error that I could not get rid of: https://github.com/tenancy/multi-tenant/issues/872

Maybe today there are better alternatives and easier to implement. 

Best,
Pedro
